File: DAY 1/question3.c
//WAP to find factorial of a no.
#include<stdio.h>
int main(){
    int i,n,fact =1;
    printf("enter the no. to find factorial of:");
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        fact*=i;
    }
    printf("factoeial of given no. is:%d",fact);
    return 0;
}

File: java_assign/DAY1/QUESTION3.java
//WAP to find factorial
package DAY1;
import java.util.Scanner;
public class QUESTION3 {
    public static void main(String[] args) {
        System.out.println("enter a no.to find factorial");
        Scanner Sc = new Scanner(System.in);
        int n =Sc.nextInt();
        int factorial =1;
        for(int i =1;i<=n;i++){
            factorial*=i;
        }
        System.out.println(factorial);
    }
    
}
